migrator/migrations/m_169_to_m_170_create_policy_categories_and_edges/migration.go

```diff
@@ -0,0 +1,138 @@
+package m169tom170
```

Comment:
@keyallis  already merged `m169tom170`.  You will need to become `m170tom171` unless @charmik-redhat pushes his before you do.  Due to the name differences of the packages these will not show as merge conflicts.